src/main/java/art/heredium/controller/user/UserCouponController.java

```diff
@@ -35,6 +36,12 @@ public ResponseEntity<List<CouponResponseDto>> getCouponsWithUsageByAccountId()
     return ResponseEntity.ok(couponResponseDtos);
   }
 
+  @GetMapping("/{coupon-uuid}")
```

Comment:
```suggestion
  @GetMapping("usage/{coupon-uuid}")
```
Tôi nghĩ thêm cái này cho clear để đỡ lẫn với coupon thường.

Reply:
Tôi đã sửa: 0afb6cb274a2930c8fca1ffe53438f122416c979